src/test/java/seedu/address/logic/parser/LibTaskParserTest.java

```diff
@@ -133,4 +136,24 @@ public void storePreviousCommand() {
         parserAddCommands.storePreviousCommand("list");
         assertEquals("list", parserAddCommands.getPreviousCommand());
     }
+
+    @Test
+    public void parseCommand_borrow() throws Exception {
+        String returnDateString = "28-Feb-2022";
+        BorrowCommand command = (BorrowCommand) parser.parseCommand(
+                 Command.BORROW_COMMAND_WORD + " "
+                        + INDEX_FIRST_PATRON.getOneBased() + " "
+                        + INDEX_FIRST_BOOK.getOneBased() + " "
+                        + returnDateString);
+        assertEquals(new BorrowCommand(INDEX_FIRST_PATRON, INDEX_FIRST_BOOK, returnDateString), command);
+    }
+
+    @Test
+    public void parseCommand_invalidReturnDate_throwsParseException() {
+        String invalidDateString = "28Feb-2022";
+        String commandString = Command.BORROW_COMMAND_WORD + " " + INDEX_FIRST_PATRON.getOneBased() + " "
+                + INDEX_FIRST_BOOK.getOneBased() + " " + invalidDateString;
+        assertThrows(ParseException.class, String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                        BorrowCommand.MESSAGE_USAGE), () -> parser.parseCommand(commandString));
```

Comment:
Good addition of test case to improve code coverage! Another test case we can consider is those with an invalid year like ```String invalidDateString = "28-Feb-99999";```.